spec/controllers/internal/v1x0/notify_controller_spec.rb

```diff
@@ -1,4 +1,5 @@
-describe Internal::V1x0::NotifyController, :type => :request do
+describe Internal::V1x0::NotifyController, :type => [:request, :v1] do
+  let(:api_version) { "internal/v1.0" }
```

Comment:
This `api_version` is here to override the default from the support file.

Reply:
If that's the case, do you even need to include the `:v1` in the `:type` list?